winmake.ps1

```diff
@@ -279,7 +279,7 @@ switch ($target) {
         if ($args.Count -gt 1) {
             $ref = $args[1]
         }
-        Win-SSHProxy -Ref $ref
+        Win-SSHProxy($ref)
```

Comment:
Not sure what this change does, was this broken before?

Reply:
yes, the `-Ref` flag wasn't returning the expected result.